src/WrappedMToken.sol

```diff
@@ -490,12 +521,53 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
         emit Claimed(account_, claimRecipient_, yield_);
```

Comment:
More of a business question and arguably not applicable since all these transfers happen within a single transaction, but it may be preferable for users to not "receive" yield that is going to the earn manager as a fee from a tax perspective.